pkgs/top-level/all-packages.nix

```
@@ -26295,7 +26295,9 @@ in
 
   xib2nib = callPackage ../development/tools/xib2nib {};
 
-  linode-cli = callPackage ../tools/virtualization/linode-cli { };
+  linode-cli = callPackage ../tools/virtualization/linode-cli {
+    inherit (python3Packages) buildPythonApplication terminaltables colorclass requests pyyaml setuptools;
+  };
```

python3Packages is it's own packages set
```suggestion
  linode-cli = python3Packages.callPackage ../tools/virtualization/linode-cli {  };
```